RESUME/api_views.py

```
@@ -17,365 +20,316 @@
+        payload = request.data
+        payload['user'] = self.authenticated_user.id
+        payload['resume_section'] = new_resume_section.id
+        education_ser: EducationUpsertSerializer = EducationUpsertSerializer(data=payload)
+        if education_ser.is_valid():
+            new_education = education_ser.save()
+            return Response(EducationFullSerializer(new_education).data, status=status.HTTP_201_CREATED)
+        else:
+            if new_resume_section:
+                new_resume_section.delete()
+            return ErrorResponse(
+                code=ErrorCode.INVALID_REQUEST, message=f'Invalid Data provided!',
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)